app/bundles/CoreBundle/Form/Type/SortableValueLabelListType.php

```diff
@@ -45,4 +64,14 @@ public function buildView(FormView $view, FormInterface $form, array $options):
         $view->vars['preaddon']      = $options['attr']['preaddon'] ?? [];
         $view->vars['postaddon']     = $options['attr']['postaddon'] ?? [];
     }
+
+    private function generateSlugFromLabel(string $label): string
+    {
+        $transliterated = InputHelper::transliterate($label);
+        $alphanumeric   = InputHelper::alphanum($transliterated, false, '_');
+        $trimmed        = trim($alphanumeric, '_');
+        $normalized     = preg_replace('/_+/', '_', $trimmed);
+
+        return strtolower($normalized ?? '');
+    }
```

Comment:
Any reason why https://github.com/mautic/mautic/blob/915b49c5dee5b905e7f8b8c831860de08a55731f/app/bundles/CoreBundle/Helper/UrlHelper.php#L299 couldn't be used?